recipes/hiredis/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+  "0.14.1":
```

Comment:
```suggestion
  "all":
```

Reply:
In the next version cmake will be used as build system, therefore the current version will be something special and I thought a separate folder for this version would fit it best.